rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1487,7 +1424,7 @@ def _remove_blockchain_accounts_test_start(
         )
         with ExitStack() as stack:
             setup.enter_blockchain_patches(stack)
-            response = requests.get(api_url_for(
+            requests.get(api_url_for(
                 api_server,
                 "blockchainbalancesresource",
```

Comment:
Small comment about styling. Almost everywhere in these test files double quotes are used and sometimes after copy-pasting I forget to change `"` to `'`. It's a bit difficult to track since if I change `"` to `'` everywhere, a lot of extra lines will be included in the PR.

Reply:
I know. Don't do it. Eventually we may find a tool (more customizable than black to do it).

Just pay attention when you touch code to not blindly copy paste.